scaleway-javascript/index.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+const pulumi = require("@pulumi/pulumi");
+const scaleway = require("@pulumiverse/scaleway");
```

Comment:
Package should be `@lbrlabs/pulumi-scaleway`

https://www.npmjs.com/package/@lbrlabs/pulumi-scaleway